recipe/meta.yaml

```diff
@@ -12,31 +12,30 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 5
```

Comment:
You can bump it to `2`. No need to skip to `5`.
It does not matter much but some users may wonder where are builds 2-4